### PR TITLE
docs(checkbox): cleanup images

### PR DESCRIPTION
**Pull request by technabors**
Updates to the Layout and images used on the checkbox documentation.

- Updated the layout to have hero and when to use sections side-by-side
- Best practices moved to row after the ^ above section.
- Labeling section moved to the Specifications section as an H4
- Images recreated (I messed these up originally and included the containing shape in the SVGs) and uploaded.
- Images placed in correct directory structure.

This doc has already had a content Audit through InfoDev, just need the markdown and images reviewed.

### CODE REVIEW

File: _component_design/checkboxes.md [hunk 1]
```diff
@@ -182,7 +189,7 @@ for more information.
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"When selecting multiple items from a table, an indeterminate
  checkbox may be used to show that several, but not all are selected."] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 2]
```diff
@@ -165,14 +172,14 @@ highlight all checkboxes in an error group.
...
 {% endcolumn %}
 </div>
 
 <div class="hxRow">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-## Indeterminate
+#### Indeterminate
 
 The indeterminate state is only used with bulk selections to indicate that one
 or more, but not all options, have been selected. Refer to the tables page
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: linked

[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 3]
```diff
@@ -149,14 +156,14 @@ alignment with the text. Asterisk floats in the margin.
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [caption:"Designating a requied checkbox field"] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 4]
```diff
@@ -114,14 +121,14 @@ and checkbox labels.
...
  {% endfigure %}
 {% endcolumn %}
 </div>
 
 <div class="hxRow">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-## States
+### States
 
 Checkbox states include the following:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: Now reads, "The checkbox component may be in the following states:"

File: _component_design/checkboxes.md [hunk 5]
```diff
@@ -67,42 +61,55 @@ allow the user to select zero, one, or several items.
...
+* Use positive and active wording for checkbox labels. Avoid negations such
+  as, "Don't send me more email". This would mean that the user would have
+  to check the box in order for something not to happen.
+* Write checkbox labels so that users know both what will happen if they
+  check a particular box, and what will happen if they leave it unchecked.
+  If you are unable to do this, we recommend using two radio buttons: one
+  for having the feature on, and one for having it off. Ensure you write
+  clear labels for each of the two cases.
+
+* **Note:** Allow users to select an option by clicking on either the button, the box
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 6]
```diff
@@ -67,42 +61,55 @@ allow the user to select zero, one, or several items.
...
 A checkbox group consists of a section label, followed pairs of checkboxes
 and checkbox labels.
+
+#### Labeling checkboxes
+
+* Use positive and active wording for checkbox labels. Avoid negations such
+  as, "Don't send me more email". This would mean that the user would have
+  to check the box in order for something not to happen.
+* Write checkbox labels so that users know both what will happen if they
+  check a particular box, and what will happen if they leave it unchecked.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 7]
```diff
@@ -67,42 +61,55 @@ allow the user to select zero, one, or several items.
...
-## Default layout
+### Default layout
 
 A checkbox group consists of a section label, followed pairs of checkboxes
 and checkbox labels.
+
+#### Labeling checkboxes
+
+* Use positive and active wording for checkbox labels. Avoid negations such
+  as, "Don't send me more email". This would mean that the user would have
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 8]
```diff
@@ -67,42 +61,55 @@ allow the user to select zero, one, or several items.
...
 
-# Specifications
+## Specifications
 
 <div class="hxRow">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-## Default layout
+### Default layout
 
 A checkbox group consists of a section label, followed pairs of checkboxes
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 9]
```diff
@@ -14,18 +14,11 @@ last-modified: 2017-08-17
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 10]
```diff
@@ -14,18 +14,11 @@ last-modified: 2017-08-17
...
-allow the user to select zero, one, or several items.
-
-{% figure [caption:"Default image"] [class:"image bg-light border"] %}
- ![]({{site.url}}/assets/images/components/content-areas/checkboxes/checkbox-hero-image.svg){:width="80%"}
- {% endfigure %}
-
-## When to use checkboxes
+### When to use checkboxes
 
 * You can use checkable input to select any number of options;
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 11]
```diff
@@ -37,7 +30,19 @@ allow the user to select zero, one, or several items.
 * If there are multiple choices and only one option can be selected, use a
   radio button or drop-down list.
 
-## Best practices for checkboxes
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: caption added.

File: _component_design/checkboxes.md [hunk 12]
```diff
@@ -48,17 +53,6 @@ allow the user to select zero, one, or several items.
 * Typically, checkboxes default to having no options selected. However,
   if Rackspace recommended an option, make that the default.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated

File: _component_design/checkboxes.md [hunk 13]
```diff
@@ -37,7 +30,19 @@ allow the user to select zero, one, or several items.
...
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+### Best practices for checkboxes
 
 * Each input is independent of all other input in the list and checking one
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md [hunk 14]
```diff
@@ -67,42 +61,55 @@ allow the user to select zero, one, or several items.
...
+### Problems solved by checkboxes
 
-* Allow users to select an option by clicking on either the button, the box
-  itself or the label; a bigger target is faster to click.
+When a user needs to make a selection in a form:
+
+- A checkbox list enables them to select one or more items while being able to visually compare them all at once.
+- A singular checkbox enables them to enable or disable a feature.
+
+When a user needs to manage a group of items:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: Now reads as a single sentence, "When a user needs to manage a group of items, checkboxes may be added to a list to allow for bulk actions."

File: _component_design/checkboxes.md [hunk 15]
```diff
@@ -67,42 +61,55 @@ allow the user to select zero, one, or several items.
   you to violate this guideline, but do try to keep choices visible whenever
   possible.
 
-### Interaction with checkboxes
+### Problems solved by checkboxes
 
-* Allow users to select an option by clicking on either the button, the box
-  itself or the label; a bigger target is faster to click.
+When a user needs to make a selection in a form:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: updated to


Checkboxes alllow users needs to make a selection in a form:

- A checkbox list enables them to select one or more items while being able to visually compare them all at once.
- A singular checkbox enables them to enable or disable a feature.

File: _component_design/checkboxes.md [hunk 16]
```diff
@@ -132,14 +139,14 @@ Checkbox states include the following:
...
 {% endcolumn %}
 </div>
 
 <div class="hxRow">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-## Required group
+#### Required group
 
 Use a red asterisk to designate that a selection is required for a
 group of checkboxes. The label should remain in vertical
```
[the remark this answers is not in the document]
Comment (technabors) [reply _component_design/checkboxes.md]: Updated.

File: _component_design/checkboxes.md [hunk 17]
```diff
@@ -67,42 +61,51 @@ allow the user to select zero, one, or several items.
   you to violate this guideline, but do try to keep choices visible whenever
   possible.
 
-### Interaction with checkboxes
+### Problems solved by checkboxes
 
-* Allow users to select an option by clicking on either the button, the box
-  itself or the label; a bigger target is faster to click.
+Checkboxes alllow users needs to make a selection in a form:
```
Comment (robb-romans) [inline _component_design/checkboxes.md]: Typos/grammar: Checkboxes allow users to make selections in a form, as follows:

File: _component_design/checkboxes.md [hunk 18]
```diff
@@ -67,42 +61,51 @@ allow the user to select zero, one, or several items.
...
 
-## Problems solved by checkboxes
+- A checkbox list enables them to select one or more items while being able to visually compare them all at once.
```
Comment (robb-romans) [inline _component_design/checkboxes.md]: s/them/users/g

File: _component_design/checkboxes.md [hunk 19]
```diff
@@ -67,42 +61,51 @@ allow the user to select zero, one, or several items.
...
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-## Default layout
+### Default layout
+
+A checkbox group consists of a section label followed by pairs of checkboxes and checkbox labels.
+
+#### Labeling checkboxes
+
+* Use positive and active wording for checkbox labels. Avoid negations such
+  as "Don't send me more email". Negative labels on checkboxes would mean that the user would have
```
Comment (robb-romans) [inline _component_design/checkboxes.md]: repaginate sentence

File: _component_design/checkboxes.md (hunk 18 above)
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

File: _component_design/checkboxes.md (hunk 17 above)
Comment (technabors) [reply _component_design/checkboxes.md]: updated.

**Pull request merged**